### ISSUE TITLE
Exception on other.js > getColumn

### ISSUE THREAD

**Issue by frantic0** [vaheqelyan/svelte-grid#77]
Hi @vaheqelyan 

I this found a run time exception which impedes loading the items on the svelte-grid.

![Screen Shot 2021-03-01 at 14 48 45 (2)](https://user-images.githubusercontent.com/2233334/109514266-be887480-7a9d-11eb-8a61-eb04b573537b.png)

This happens with svelte-grid version 3.6.5 and 4.0.0 (just upgraded and tested) when I resize the window with a svelte-grid component loaded on the UI.   

(https://github.com/frantic0/sema-next/blob/editors/src/pages/playground/index.svelte)[https://github.com/frantic0/sema-next/blob/editors/src/pages/playground/index.svelte]

**Comment by vaheqelyan** [comment]
I am on mobile, can't access the file

**Comment by frantic0** [comment]
are you referring to the image that I posted?

**Comment by vaheqelyan** [comment]
nope, I mean the link (https://github.com/frantic0/sema-next/blob/editors/src/pages/playground/index.svelte) [https://github.com/frantic0/sema-next/blob /editors/src/pages/playground/index.svelte] which you included in the description

**Comment by vaheqelyan** [comment]
I mean, if a value cannot be iterated, then it must be `undefined`. what the columns are equal to, in the debug session ?

**Comment by frantic0** [comment]
you're right, it is undefined. I think this is related to https://github.com/vaheqelyan/svelte-grid/issues/78, where if we don't express a cols config ([resolution, columns])) and resize a window to exceed that resolution that exception pops-up
